payment_term.py

```diff
@@ -31,19 +31,6 @@ def __setup__(cls):
         super(PaymentTerm, cls).__setup__()
         cls._order.insert(0, ('name', 'ASC'))
 
-    @classmethod
-    def validate(cls, terms):
-        super(PaymentTerm, cls).validate(terms)
-        for term in terms:
-            term.check_remainder()
-
-    def check_remainder(self):
-        if not self.lines or not self.lines[-1].type == 'remainder':
```

Comment:
juste if not self.lines: return n'était pas suffisant ? (comme ça on garderait qd même l'assurance d'avoir un reste en dernier)

Reply:
Pour moi il faut que compute ajoute automatiquement le reste à payer à la date.

Reply:
@cedk  Je ne suis pas sur de bien comprendre: Les tests passent en l'état sur le module, quelle est l'idée derrière d'ajouter automatiquement la ligne de remainder en date ?

Reply:
Au temps pour moi, ce cas la n'est pas testé dans les scénarios, of course.

Reply:
Quelque chose en ce sens j'imagine https://github.com/coopengo/account_invoice/pull/13/commits/d990dcf6128fe20a670511082d7410e3427a0696